### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by tenshi0xx** [Lost-MSth/Arcaea-server#151] [Q&A] [answered]
How do i import arcsong.db to arcaea_database.db

**Comment by Lost-MSth** [comment]
https://github.com/Lost-MSth/Arcaea-server/wiki/Instruction-for-use#import-song-data

**Reply by tenshi0xx** [reply, marked as answer]
no, that's for manually import song, i want to merge song from arcsong.db to arcaea_database.db

**Reply by Lost-MSth** [reply]
Please look at the doc carefully.

> Since v2.9, the song database arcsong.db will merge into arcaea_database.db, and the initial data is no longer provided. You can use update_song.py under the tools folder to do data migration or import.
